examples/light-switch-app/efr32/include/FreeRTOSConfig.h

```diff
@@ -0,0 +1,303 @@
+/*
```

Comment:
This file is outdated for power_management/SED build

Reply:
done